play/type-shooter/js/app.js

```diff
@@ -3,34 +3,97 @@ let words = [
 ];
             
 let wordDiffLevel;
-let currentWord;
+let difficulty = "medium";
+//let currentWord = undefined;
+let currentWords = [];
 let balloonsPopped = 0;
+let spawnInterval = 3000;
+let animationSpeed = 5000;
 const sectors = ["sector_one", "sector_two", "sector_three"];
-            
-// Shuffle words list
-words = shuffle(words);
-            
-// Spawn a new balloon every three seconds
-setInterval(spawnBalloon(), 3000);
-            
-const el = document.getElementById('input');
 
-// always be focused on the input
-el.focus();
-el.onblur = () => setTimeout(() => el.focus());
+const EASY_MULTIPLIER = 1;
+const MEDIUM_MULTIPLIER = 1.25;
+const HARD_MULTIPLIER = 1.5;
+
+(function initUI() {
+    // show instructions modal
+    $("#myModal").modal("show");
+})();
+
+/**
+ * Starts the Type Shooter game loop
+ */
+function startGame() {
+
+    document.getElementById("balloonShooter").style.display = "";
+    document.getElementById('input').disable = false;
+    document.getElementById('input').focus();
+
+    // Shuffle words list
+    words = shuffle(words);
+
+    spawnBalloon();
+
+    // Spawn a new balloon every spawnInterval / difficulty multiplier seconds
+    setInterval(() => {spawnBalloon();}, spawnInterval / (
```

Comment:
Perhaps make this into a function that returns the multiplier or the interval just to make this code less complex.